src/quiz/Description.jsx

```diff
@@ -39,7 +39,7 @@ export default function Description() {
     // TODO: APIとの通信
     for (let i = 0; i < data.playerId[0].ngWord.length; i++) {
       setNgWord((ngWord) => ngWord + data.playerId[0].ngWord[i]);
-      if (i != data.playerId[0].ngWord.length - 1)
+      if (i !== data.playerId[0].ngWord.length - 1)
         setNgWord((ngWord) => ngWord + ", ");
     }
```

Comment:
```suggestion
    }
    // eslint-disable-next-line react-hooks/exhaustive-deps
```

こんなコメントを追加することでeslintのwarnを回避できます(使うときがあればその上の行に理由を書いておいてほしいかも)

Reply:
なるほど
覚えておきます！